os/binfmt/libelf/libelf_sections.c

```diff
@@ -194,12 +194,15 @@ void elf_save_bin_section_addr(struct binary_s *bin)
 	if (bin_info != NULL) {
 		strncpy(bin_info->bin_name, bin->bin_name, BIN_NAME_MAX);
 		bin_info->text_addr = (uint32_t)bin->alloc[ALLOC_TEXT];
+
+		binfo("[%s] text_addr : %x", bin_info->bin_name, bin_info->text_addr);
+#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
```

Comment:
You'd better add new line between line 205 & 206, otherwise this will mixed with next log if CONFIG_OPTIMIZE_APP_RELOAD_TIME is disabled.
But If I were you, I print out line per item
text addr : 0x12312313
rodata_adr : 0x15120481412
...

Reply:
Done. Thanks.